webdriver/tests/execute_async_script/execute_async.py

```diff
@@ -26,6 +26,82 @@ def test_no_browsing_context(session, closed_window):
     assert_error(response, "no such window")
 
 
+@pytest.mark.parametrize("values", [
+      {"primitive": "null", "expected": None},
+      {"primitive": "undefined", "expected": None},
+      {"primitive": "true", "expected": True},
+      {"primitive": "false", "expected": False},
+      {"primitive": "23", "expected": 23},
+      {"primitive": "'a string'", "expected": "a string"}
+    ])
+def test_primitive_serialization(session, values):
+    response = execute_async_script(
+        session, "arguments[0](%s);" % values["primitive"]
+    )
+    assert_success(response)
+    assert response.body["value"] == values["expected"]
+
+
+# > To clone an object, taking the arguments value, seen, and clone algorithm:
+# >
+# > 1. Let result be the value of the first matching statement, matching on
+# > value:
+# >
+# >     instance of NodeList
+# >     instance of HTMLCollection
+# >     instance of Array
+# >         A new Array which length property is equal to the result of getting
+# >         the property length of value.
+# >     Otherwise
+# >         A new Object.
+#
+# https://w3c.github.io/webdriver/#dfn-clone-an-object
```

Comment:
Please leave out all quotes of the specification. We removed most of these some time ago because we weren’t able to keep them up with the changing spec.